src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/AsyncHelper.cs

```diff
@@ -0,0 +1,38 @@
+// <copyright file="AsyncHelper.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation
+{
+    /// <summary>
+    /// Class used for calling asynchronous methods from synchronous ones.
+    /// </summary>
+    internal class AsyncHelper
+    {
+        private static readonly TaskFactory TaskFactory = new
+            TaskFactory(CancellationToken.None, TaskCreationOptions.None, TaskContinuationOptions.None, TaskScheduler.Default);
+
+        public static TResult RunSync<TResult>(Func<Task<TResult>> func)
+            => TaskFactory
+                .StartNew(func)
+                .Unwrap()
+                .GetAwaiter()
+                .GetResult();
```

Comment:
@rypdal Could you explain a bit why `AsyncHelper` is needed?

Reply:
Since the `BaseExporter<T>` class has only sync version of `Export` method, but before NET 5.0 the `HttpClient` had only async version of Send method (`SendAsync`). So, we have to call async method synchronously (similar case is in [ZipkinExporter](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Zipkin/ZipkinExporter.cs#L76)). Another question is: why should we keep `AsyncHelper` for only a single case where this call is required  ?  I think this class can be removed.

Reply:
Agree makes sense to remove and follow similar pattern as the Zipkin exporter if that meets the need.

Reply:
Completely removed the `AsyncHelper` class. Used code similar to [Zipkin exporter](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Zipkin/ZipkinExporter.cs#L76) for calling `HttpClient.SendAsync(...)`.